src/contrib/loggers/Akka.Logger.SemanticSerilog/LogParser.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using Sprache;
+
+namespace Akka.Logger.SemanticSerilog
+{
+    internal static class LogParser
+    {
+        public static Parser<StringFormat> StringFormat =
```

Comment:
Does the Sprache is really a justified dependency in this case? Doesn't this feel like taking a canon to kill a mosquito?